### ISSUE TITLE
iOS: deprecation warnings 

### ISSUE THREAD

**Issue by Dejal** [samuelclay/NewsBlur#978]
Building the iOS app results in a number of deprecation warnings, for `UIAlertView`, `UIPopoverController`, `openURL:`, `CFReadStreamCreateForStreamedHTTPRequest`, and others.

It'd be a good idea to update these to modern APIs sometime.  I'd be happy to help with that if you wish.

**Comment by samuelclay** [comment]
Yep, I'm on these in the ios8 branch. Gonna require removing ASI and moving to AFNetworking. We already have AFNetworking, but some requests are still on the old ASI.
